cli/cmd/physical_design/optimize.hpp

```diff
@@ -44,6 +44,7 @@ class optimize_command : public command
         add_option("--max_gate_relocations,-m", max_gate_relocations,
                    "Specify the maximum number of relocations to try for each gate (defaults "
                    "to the number of tiles in the layout).");
+        add_flag("--planar_optimization,-p", ps.planar_optimization, "Enable planar optimization");
```

Comment:
Can you elaborate a little here on what that entails? A user could assume the resulting layout will be planar. But what you do is solely not adding crossings (and maybe reducing them) but not guaranteeing planarity unless the input was already planar.